.pre-commit-config.yaml

```
@@ -0,0 +1,56 @@
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+repos:
+- repo: https://github.com/pre-commit/pre-commit-hooks
+  rev: v4.1.0
+  hooks:
+  - id: check-added-large-files
+  - id: check-toml
+  - id: check-yaml
+  - id: end-of-file-fixer
+  - id: trailing-whitespace
+- repo: https://github.com/psf/black
+  rev: 22.1.0
+  hooks:
+  - id: black
+# TODO: Remove comments when type-hinting is implemented
+#- repo: https://github.com/pre-commit/mirrors-mypy
+#  rev: v0.931
+#  hooks:
+#  - id: mypy
+#    exclude: ^(docs/|example-plugin/|tests/fixtures)
+# TODO: Remove comments when/if poetry is implemented, AND when/if pytest is implemented
+#- repo: local
+#  hooks:
+#  - id: pytest
+#    name: Check pytest unit tests pass
+#    entry: poetry run duty test
+#    pass_filenames: false
+#    language: system
+#    types: [python]
+- repo: https://github.com/macisamuele/language-formatters-pre-commit-hooks
+  rev: v2.2.0
```

```suggestion
  rev: v2.4.0
```
Update to the most recent version